packages/aura-language-server/package.json

```diff
@@ -41,26 +41,27 @@
     "vscode-uri": "1.0.6"
   },
   "devDependencies": {
+    "@jest/reporters": "^29.5.0",
     "@types/babel-types": "^7.0.8",
     "@types/fs-extra": "^5.0.4",
     "@types/glob": "^7.1.3",
-    "@types/jest": "^26.0.10",
+    "@types/jest": "^29.5.0",
     "@types/mock-fs": "^4.10.0",
     "@types/node": "^16.0.0",
     "@types/shelljs": "^0.8.8",
     "babel-types": "^6.26.0",
     "eslint": "^7.7.0",
     "glob": "^7.1.6",
     "husky": "^4.2.5",
-    "jest": "^26.4.1",
+    "jest": "^29.5.0",
     "jest-extended": "^0.11.5",
     "lint-staged": "^10.2.11",
     "mock-fs": "^4.11.0",
     "npm-run": "^5.0.1",
     "prettier": "^2.0.5",
     "shelljs": "^0.8.4",
-    "ts-jest": "^26.2.0",
+    "ts-jest": "^29.1.0",
     "tsc-transpile-only": "0.0.3",
-    "typescript": "3.7.5"
+    "typescript": "5.0.4"
```

Comment:
Dependency bumps here and `lightning-lsp-common` were necessary due to the top-level TypeScript upgrade to v5.